### PR TITLE
[4] Tabbed process list dark theme preview (tabs only, no forms)

### PR DESCRIPTION
**Pull request by JulianWielga**

### CODE REVIEW

File: ui/client/components/EspButton.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import cn from "classnames"
+import React from "react"
+import styles from "../stylesheets/graph.styl"
+import {ButtonWithFocus} from "./withFocus"
+
+export function EspButton({className, ...props}: React.DetailedHTMLProps<React.ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>) {
```
Comment (lciolecki) [inline ui/client/components/EspButton.tsx:6]: ESP is old name. Please change it to NKButton.

File: ui/client/components/AddProcessDialog.tsx [hunk 2]
```diff
@@ -85,7 +86,7 @@ class AddProcessDialog extends React.Component<Props, State> {
                     <div className="node-row">
                       <div className="node-label">Name</div>
                       <div className="node-value">
-                        <input
+                        <InputWithFocus
```
Comment (lciolecki) [inline ui/client/components/AddProcessDialog.tsx:89]: Do we have Input/Select/ etc WithoutFocus? Can we consider just Input or NussknackerInput? I assume that in future we want extract base componnents to secondary liby.

File: ui/client/components/MenuBar.tsx [hunk 3]
```diff
@@ -9,12 +9,13 @@ import {ProcessesTabData} from "../containers/Processes"
...
 import {Signals} from "../containers/Signals"
 import {Flex} from "./common/Flex"
+import {ButtonWithFocus} from "./withFocus"
 
 function useStateWithRevertTimeout<T>(startValue: T, time = 10000): [T, React.Dispatch<React.SetStateAction<T>>] {
   const [defaultValue] = useState<T>(startValue)
   const [value, setValue] = useState<T>(defaultValue)
   useEffect(() => {
-    let t: NodeJS.Timeout
+    let t
```
Comment (lciolecki) [inline ui/client/components/MenuBar.tsx:18]: Please rename it to more obvious like timeout?

File: ui/client/components/QueriedStateTable.js [hunk 4]
```diff
@@ -34,29 +35,31 @@ class QueriedStateTable extends React.Component {
...
           <div className="esp-form-row">
             <p>Query name</p>
-            <select
+            <SelectWithFocus
               value={this.selectedQueryName()}
               onChange={(e) => this.setState({queryName: e.target.value, processId: this.processesForQueryName(e.target.value)[0]})}
             >
-              {_.keys(this.props.availableQueryableStates).map((queryName, index) => (<option key={index} value={queryName}>{queryName}</option>))}
-            </select>
+              {_.keys(this.props.availableQueryableStates).map((queryName, index) => (
```
Comment (lciolecki) [inline ui/client/components/QueriedStateTable.js:42]: I know it's not part of your change.. But can we import from lodash only this functions with we use?

File: ui/client/components/healthCheck.styl [hunk 5]
```diff
@@ -0,0 +1,19 @@
+@require "../stylesheets/_variables.styl"
```
Comment (lciolecki) [inline ui/client/components/healthCheck.styl:1]: There is some problem with formatting? tabs / 4 spaces? Evrywere at FE we use 2 spaces.

File: ui/client/components/table/searchIcon.styl [hunk 6]
```diff
@@ -0,0 +1,24 @@
+@require "../../stylesheets/_variables.styl"
```
Comment (lciolecki) [inline ui/client/components/table/searchIcon.styl:1]: Formatting?

File: ui/client/components/withFocus.styl [hunk 7]
```diff
@@ -0,0 +1,7 @@
+@require "../stylesheets/_variables.styl"
```
Comment (lciolecki) [inline ui/client/components/withFocus.styl:1]: Formatting?

File: ui/client/components/AddProcessDialog.tsx (hunk 2 above)
Comment (JulianWielga) [reply ui/client/components/AddProcessDialog.tsx:89]: those components are temporarily needed to separate styles

File: ui/client/components/QueriedStateTable.js (hunk 4 above)
Comment (JulianWielga) [reply ui/client/components/QueriedStateTable.js:42]: bring nothing, could introduce errors, there are no tests or ts.

File: ui/client/components/healthCheck.styl (hunk 5 above)
Comment (JulianWielga) [reply ui/client/components/healthCheck.styl:1]: its not formatting, stylus is indent dependent, tabs are intentional, one level - one char. 2,3,4,5,5,156.... spaces = problems.

**Pull request merged**